chapters/en/chapter13/2.mdx

```diff
@@ -0,0 +1,254 @@
+
+From a general perspective, let's look at the benefits of using RL in LLMs:
+
+| Benefit | Description |
+|---------|-------------|
+| Improved Control | RL allows us to have more control over the kind of text LLMs generate. We can guide them to produce text that is more aligned with specific goals, like being helpful, creative, or concise. |
+| Enhanced Alignment with Human Values | RLHF, in particular, helps us align LLMs with complex and often subjective human preferences. It's hard to write down rules for "what makes a good answer," but humans can easily judge and compare responses. RLHF lets the model learn from these human judgments. |
+| Mitigating Undesirable Behaviors | RL can be used to reduce negative behaviors in LLMs, such as generating toxic language, spreading misinformation, or exhibiting biases. By designing rewards that penalize these behaviors, we can nudge the model to avoid them. |
+
+Reinforcement Learning from Human Feedback has been used to train many of the most popular LLMs today, such as OpenAI's GPT-4, Google's Gemini, and DeepSeek's R1. There are a wide range of techniques for RLHF, with varying degrees of complexity and sophistication. In this chapter, we will focus on Generalized Policy Rectification Optimization (GRPO), which is a technique for RLHF that has been shown to be effective at training LLMs that are helpful, harmless, and aligned with human preferences.
+
+## Why should we care about GRPO (Generalized Policy Rectification Optimization)?
+
+There are many techniques for RLHF but this course is focused on GRPO because it represents a significant advancement in reinforcement learning for language models.
+
+Let's briefly consider two of other popular techniques for RLHF:
+
+- Proximal Policy Optimization (PPO)
+- Direct Preference Optimization (DPO)
+
+Proximal Policy Optimization (PPO) was one of the first highly effective techniques for RLHF. It uses a policy gradient method to update the policy based on the reward from a separate reward model.
+
+Direct Preference Optimization (DPO) was later developed as a simpler technique that eliminates the need for a separate reward model using preference data directly. Essentially, framing the problem as a classification task between the chosen and rejected responses.
+
+<Tip>
+
+DPO and PPO are complex reinforcement learning algorithms in their own right, which we will not cover in this course. If you're interested in learning more about them, you can check out the following resources:
+
+- [Proximal Policy Optimization](https://huggingface.co/docs/trl/main/en/ppo_trainer)
+- [Direct Preference Optimization](https://huggingface.co/docs/trl/main/en/dpo_trainer)
+
+</Tip>
+
+Unlike DPO and PPO, GRPO groups similar samples together and compares them as a group. The group-based approach provides more stable gradients and better convergence properties compared to other methods.
```

Comment:
I don't know if it's said later, but here I think it's important to mention that in GRPO, we don't have the stages that were introduces earlier "collect human preferences" and "train a reward model"

Reply:
ok it's literally this next line 😂